makefiles/pseudomodules.inc.mk

```diff
@@ -17,6 +17,7 @@ PSEUDOMODULES += gnrc_sixlowpan_router
 PSEUDOMODULES += gnrc_sixlowpan_router_default
 PSEUDOMODULES += gnrc_sock_check_reuse
 PSEUDOMODULES += gnrc_txtsnd
+PSEUDOMODULES += l2filter_%
```

Comment:
So... `l2filter_foobar` gives me blacklist... Can you give explicit names here. Just having this translate to anything doesn't seem like a good idea. Pseudo-modules are already in-transparent as they are.